src/BenchmarkDotNet/Exporters/PrettyGithubMarkdownDiffDisassemblyExporter.cs

```diff
@@ -0,0 +1,136 @@
+                .Where(results.ContainsKey).ToList();
+
+            logger.WriteLine($"## {summary.Title}");
+
+            for (int i = 0; i < benchmarksCases.Count; i++)
+            {
+                var firstBenchmarkCase = benchmarksCases[i];
+                for (int j = i + 1; j < benchmarksCases.Count; j++)
+                {
+                    var secondBenchmarkCase = benchmarksCases[j];
+
+                    var firstFileName = Export(summary, results[firstBenchmarkCase]);
+                    var secondFileName = Export(summary, results[secondBenchmarkCase]);
+                    try
+                    {
+                        var builder = new StringBuilder();
+
+                        RunGitDiff(firstFileName, secondFileName, builder);
+
+                        if (firstBenchmarkCase.Descriptor.WorkloadMethod == secondBenchmarkCase.Descriptor.WorkloadMethod) // diff between the same method for different JITs
+                        {
+                            logger.WriteLine($"**Diff for {firstBenchmarkCase.Descriptor.WorkloadMethod.Name} method between:**");
+                            logger.WriteLine($"{GetImportantInfo(summary[firstBenchmarkCase])}");
+                            logger.WriteLine($"{GetImportantInfo(summary[secondBenchmarkCase])}");
+                        }
+                        else // different methods, same JIT
+                        {
+                            logger.WriteLine(
+                                $"**Diff between {firstBenchmarkCase.Descriptor.WorkloadMethod.Name} and {secondBenchmarkCase.Descriptor.WorkloadMethod.Name}**");
+                            logger.WriteLine($"on {GetImportantInfo(summary[firstBenchmarkCase])}.");
+                        }
+
+                        logger.WriteLine("```diff");
+                        logger.WriteLine(builder.ToString());
+                        logger.WriteLine("```");
+                    }
+                    finally
+                    {
+                        File.Delete(firstFileName);
+                        File.Delete(secondFileName);
+                    }
+                }
+            }
+        }
+
+        private static string Export(Summary summary, DisassemblyResult disassemblyResult)
+        {
+            string filePath = $"{Path.Combine(summary.ResultsDirectoryPath, Guid.NewGuid().ToString())}-diff.temp";
+
+            if (File.Exists(filePath))
+                File.Delete(filePath);
+
+            using (var stream = StreamWriter.FromPath(filePath))
+            {
+                Export(new StreamLogger(stream), disassemblyResult, quotingCode: false);
+            }
+
+            return filePath;
+        }
+
+        private static string GetImportantInfo(BenchmarkReport benchmarkReport) => benchmarkReport.GetRuntimeInfo();
+
+        private static void RunGitDiff(string firstFile, string secondFile, StringBuilder result)
+        {
+            var startInfo = new ProcessStartInfo
```

Comment:
I can reuse `ProcessHelper.RunAndReadOutputLineByLine` because `ProcessHelper.RunAndReadOutput` doesn't throw exception in case git is not installed on the system. There is a problem: https://github.com/dotnet/BenchmarkDotNet/blob/c724e9487fa8a2987712adc42c59edd0e4357d1b/src/BenchmarkDotNet/Helpers/ProcessHelper.cs#L29-L36

I can refactor ProcessHelper but I think it should be a different task.